cmd/tether/utils_linux.go

```diff
@@ -90,6 +91,23 @@ func (t *osopsLinux) SetHostname(hostname string) error {
 	return nil
 }
 
+func pciToMAC(pciAddr string) (string, error) {
+	addrPath := "/sys/bus/pci/devices/" + pciAddr + "/net/*/address"
```

Comment:
We will be aliasing interfaces but a basic test with `ip link add <cidr> brd + dev eth0 label eth0:0` shows that an IP alias doesn't show here.
@andrewtchin I would suggest using ioutil.ReadDir to get an actual path name as I'm not sure how Go would handle the * in the case of multiple entries - I'm surprised it works with one honestly but I learn something new most days :)